pylua/test/test_timer.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+
+import timeit
+import threading
+import pylua.timer as timer
+
+
+class TestPyTimer:
+
+    def _timer_task(self):
+        self._task_is_run = True
+        self._event.set()
+
+    def __init__(self):
+        self._interval = 50  # interval in milliseconds
+        self._timer = timer.PyTimer(self._timer_task, self._interval)
+        self._task_is_run = False
+        self._event = threading.Event()
+
+    def _wait_event(self, wait_time):
+        wait_time = timer.msec_to_sec(wait_time)
+        self._event.clear()
```

Comment:
@malirod Why? Event has its own timeout and as far as I understand if nobody call event.set() main thread continues to work after event timeout.